components/match_ticker/commons/match_ticker.lua

```diff
@@ -113,6 +116,13 @@ function MatchTicker:init(args)
 		enteredOpponentOnLeft = hasOpponent and Logic.readBool(args.enteredOpponentOnLeft or hasOpponent),
 		showInfoForEmptyResults = Logic.readBool(args.showInfoForEmptyResults),
 		onlyHighlightOnValue = args.onlyHighlightOnValue,
+		tiers = args.tiers and Array.filter(Array.parseCommaSeparatedString(args.tiers), function (tier)
+					local identifier = Tier.toIdentifier(tier)
+					return type(identifier) == 'number' and Tier.isValid(identifier)
+				end) or nil,
+		tierTypes = args.tiertypes and Array.filter(Array.parseCommaSeparatedString(args.tiertypes), function(tiertype)
+					return Tier.isValid(1, tiertype)
```

Comment:
Uff don't like this (the force of the first parameter) but also understand why based on the library. Let's make an issue out of it to improve it later on